recipes/newversion/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "newversion" %}
+{% set version = "1.8.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/newversion-{{ version }}.tar.gz
+  sha256: 7cc493c901717c184cc67281d29c7d9842ef90ddb055590db8bff3aa39910a82
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - newversion = newversion.main:main_cli
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - poetry
+    - pip
+  run:
+    - python >=3.6
+    - packaging >=20.0
+    - typing-extensions >=3.7.4
+
+test:
+  imports:
+    - newversion
+  commands:
+    - pip check
+    - newversion --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/vemel/newversion
```

Comment:
is this the proepr upstream? It's a fork of https://github.com/altitudenetworks/newversion

Reply:
Good question! I'm not sure.

@vemel is the sole contributor, and the altitudenetworks version points to him:
![image](https://user-images.githubusercontent.com/15216687/140314239-54ed8440-63ae-420a-8a6f-f6079641d40a.png)

@vemel, what's the story here? Is there any possibility to have altitudenetworks archive the original repo?

Reply:
Yes, I am the only contributor to the repo and it has an MIT license. `https://github.com/vemel/newversion` is a correct link.

Reply:
@vemel, this is already clear, and I don't find anything suspicious. Purely to avoid potential confusion, I'm just wondering if it's possible to add a note on the old repo that development is continuing on your fork. (If you're unable, it's fine.) Also, I notice that there is a broken link on the old repo to https://vemel.github.io/newversion/. Perhaps you could make this redirect to your new fork?

Reply:
@wolfv to be clear I'm in favor of merging. (I don't think this should be a blocker.)

Reply:
I will add this note soon. It is surely possible.